Algorithms/nth_Fibonacci_number_using_Golden_ratio.cpp

```diff
@@ -0,0 +1,41 @@
+//Fibonacci series = 0,1,1,2,3,5,8,13,21,34,........................
+//Golden Ratio = (1+sqrt(5))/2=1.6180339887
+//Golden ratio may give us incorrect answer , we can get correct result if we round up the result at each point
+// nth fibonacci number = round(n-1 th fibonacci number * golden ratio)
+//This method can calculate first 34 fibonacci number correctly after taht there may be difference from the correct value
+
+//C++ program for nth fibonacci number using golden ratio
+#include<bits/stdc++.h>
+using namespace std;
+
+double golden_ratio = 1.6180339;
+//fibonacci number upto n=5 
+int fibonacci[6]={0,1,1,2,3,5};
```

Comment:
Make it dynamic and not static

Reply:
This is the basic array which considered as base for all other number. This method only calculate the fibonacci number between 6 to 35.
Plz checkout the comments ma'am

Reply:
Okay why only 6 to 35?why not any number?